Assets/Plugins/Source/Core/Utility/FileUtility.cs

```diff
@@ -71,7 +102,46 @@ public static string GetProjectPath()
         /// <returns>The contents of the file at the indicated path as a string.</returns>
         public static string ReadAllText(string path)
         {
-            return File.ReadAllText(path);
+            string text;
+#if UNITY_ANDROID
+            using var request = UnityEngine.Networking.UnityWebRequest.Get(filePath);
+            request.timeout = 2; //seconds till timeout
+            request.SendWebRequest();
+
+            // Wait until webRequest completed
+            while (!request.isDone) { }
+
+#if UNITY_2020_1_OR_NEWER
+            if (request.result != UnityEngine.Networking.UnityWebRequest.Result.Success)
+            {
+                Debug.Log("Requesting " + filePath + ", please make sure it exists.");
+                throw new Exception("UnityWebRequest didn't succeed, Result : " + request.result);
+            }
+#else
+            if (request.isNetworkError || request.isHttpError)
+            {
+                Debug.Log("Requesting " + filePath + ", please make sure it exists and is a valid config");
+                throw new Exception("UnityWebRequest didn't succeed : Network or HTTP Error");
+            }
+#endif
+            text = request.downloadHandler.text;
+#else
+            text = File.ReadAllText(path);
```

Comment:
This is the implementation that is unique to Android regarding `File.ReadAllText`.